test/verify/check-kdump

```diff
@@ -483,5 +506,145 @@ class TestKdumpNFS(KdumpHelpers):
             raise
 
 
+@testlib.timeout(900)
+@testlib.onlyImage("Only Fedora has ansible on the image", "fedora-*")
+class TestKdumpAnsible(KdumpHelpers, packagelib.PackageCase):
```

Comment:
It would be more realistic to run ansible or the shell script on a pristine machine (i.e. a second `provision=`). In this test you already correctly configure kdump via cockpit. Similar to TestKdumpNFS, or actually use its testBasic to at least cover the NFS case.

Reply:
It is, but then we need to do a switcheroo on how we configure kdump and where we run Ansible because it needs `self.enableRepo` to function otherwise `dnf` plainly hangs.

Reply:
So what we need to do is run ansible on machine "0" and setup kdump on the second machine using Cockpit. If that is possible sure. But I don't think it is? trivially?

Reply:
Ah, looking at `check-superuser` I suppose we can do:
```
            b.enter_page("/system", host="10.111.113.2")
```